server/room-battle.js

```diff
@@ -71,11 +71,11 @@ class RoomBattlePlayer {
 	updateChannel(/** @type {User | Connection} */ user) {
 		if (user instanceof Users.Connection) {
 			// "user" is actually a connection
-			Sockets.channelMove(user.worker, this.game.id, this.slotNum % 2 + 1, user.socketid);
+			Sockets.channelMove(user.worker, this.game.id, this.slotNum + 1, user.socketid);
```

Comment:
In theory it should still be `% 2` in multi battles.

Reply:
Who's "theory"? Do you really share information in 2v2? Also, do we know the gameType here?

Reply:
I'm relatively sure they share max-HP information, which is the main reason why have the channel system in the first place.

Reply:
max-HP is one thing, but theres also a bunch of other privileged information. do we need research to figure out exactly what is shared? I guess it doesn't matter because with the code on HEAD it gets shared anyway, so this wouldnt be introducing leaks (though they might already exist). 

This only applies to 2v2 though (GameType `'multi'`), right? Can we detect that here?

Reply:
As far as I'm aware, the only privileged information is max-HP, gen-3 Pressure activation, and a bunch of hints. I'm fine with all of those being shared; that seems a much better solution than trying to separate them out into shared and unshared data.

Yes, this only applies to `'multi'`. In theory that should be `this.game.gameType` or something along those lines.

Reply:
Even more ideally, the player should track its own channel, so it doesn't need to be calculated on-the-fly here.

Reply:
Done, I think.